sale_order_line_sequence/models/sale_order_line.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2017 ForgeFlow S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    # re-defines the field to change the default
+    sequence = fields.Integer(
```

Comment:
@OriolMForgeFlow And why not avoiding the addition of a new field and changing the visible_sequence field to readonly=False and adapting the compute subsequently?

Reply:
We have decided to use a new field because the standard Odoo field was not working properly as it was modifying the values. Also, we didn't want to alter the logic of the standard sequence field.